pylint_odoo/examples/.jslintrc

```diff
@@ -185,7 +188,10 @@
     ],
     "lines-around-comment": "off",
     "max-depth": "error",
-    "max-len": "off",
+    "max-len": ["error", {
+      "code": 79,
```

Comment:
Do we want to strict also here? I don't know if JS has the same limits as PEP8

Reply:
We can set the limits we want. I'm open to suggestions, but I just wanna have a standard.

Reply:
Yeah, I'm asking if you know a convention on this for the JS part.

Reply:
There are plenty of them, we just have to choose. The most common are 80 and 120 AFAIK.

Reply:
personally I'd go 120 :)

Reply:
I'm with 80

Reply:
So what do we do? :thinking:

Reply:
OK for me also for 80

Reply:
I'll try to survive :sob: :rofl: